test/sagemaker_tests/pytorch/training/requirements.txt

```diff
@@ -15,8 +15,8 @@ pytest-rerunfailures==8.0
 pytest-xdist==1.28.0
 PyYAML==5.4
 requests==2.24
-torch==1.6.0
-torchvision==0.7.0
+torch==1.11
+torchvision==0.12
```

Comment:
Interesting that these were already pinned whereas inference wasn't. Opposite question, do we know why these are pinned?

Reply:
We copied these test requirements from the sagemaker-pytorch-training/inference-toolkit repositories. I believe this version pin was a consequence of that.